src/controls/Controls.spec.js

```diff
@@ -1 +1,28 @@
-// Test away!
\ No newline at end of file
+// Test away!
+import React from 'react';
+import * as rtl from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import Controls from './Controls';
+afterEach(rtl.cleanup);
+
+describe('Controls', () => {
+    it('should match snapshot', () => {
+        const wrapper = rtl.render(<Controls />);
+        expect(wrapper.baseElement).toMatchSnapshot();
+    })
+    it('provides buttons to toggle the closed and locked states', () => {
+        const { getByText } = rtl.render(<Controls />);
+        getByText(/Lock Gate/);
+        getByText(/Close Gate/);
+    });
+    it('the closed toggle button is disabled if the gate is locked', () => {
+        const { getByText } = rtl.render(<Controls closed={true} locked={true}/>);
+        const openButton = getByText(/Open Gate/);
+        expect(openButton.disabled).toBeTruthy();
```

Comment:
Take a look [here](https://github.com/testing-library/jest-dom#table-of-contents) for the new methods that are added to the `expect` API when you import from jest-dom.

You could simplify this to:
```js
expect(getByText('Open Gate')).toBeDisabled();
```